llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -971,10 +971,16 @@ enum NodeType {
   FMINNUM,
   FMAXNUM,
 
-  /// FMINNUM_IEEE/FMAXNUM_IEEE - Perform floating-point minimum or maximum on
-  /// two values, following the IEEE-754 2008 definition. This differs from
-  /// FMINNUM/FMAXNUM in the handling of signaling NaNs. If one input is a
-  /// signaling NaN, returns a quiet NaN.
+  /// FMINNUM_IEEE/FMAXNUM_IEEE - Perform floating-point minimumNumber or
```

Comment:
Can we make this comment and the one in `GenericOpcodes.td` identical? I feel like I'm playing spot the difference trying to review this.